cico_setup.sh

```diff
@@ -37,13 +43,13 @@ push_image() {
     image_name=$(make get-image-name)
     image_repository=$(make get-image-repository)
     short_commit=$(git rev-parse --short=7 HEAD)
-    push_registry="push.registry.devshift.net"
+    #push_registry="push.registry.devshift.net"
```

Comment:
Please leave these changes to https://github.com/fabric8-hdd/supervisor/pull/38

Reply:
I only meant that after the script is verified and the PR is ready for merge, please retain back the old state of these lines as it would later be changed as in #38 once that's merged..